src/main.zig

```diff
@@ -1417,6 +1417,12 @@ fn buildOutputType(
                         emit_bin = .{ .yes = arg["-femit-bin=".len..] };
                     } else if (mem.eql(u8, arg, "-fno-emit-bin")) {
                         emit_bin = .no;
+                    } else if (mem.eql(u8, arg, "-femit-pch")) {
+                        clang_preprocessor_mode = .pch;
+                        emit_bin = .yes_default_path;
+                    } else if (mem.startsWith(u8, arg, "-femit-pch=")) {
+                        clang_preprocessor_mode = .pch;
+                        emit_bin = .{ .yes = arg["-femit-pch=".len..] };
```

Comment:
Implementing it this way means that it's possible to write something like this:

```
zig build-exe ... -femit-pch
```

which is nonsense. Right? `zig build-pch` is needed to implement this correctly.